momento-sdk/src/main/java/momento/sdk/responses/GenerateDisposableTokenResponse.java

```diff
@@ -0,0 +1,40 @@
+package momento.sdk.responses;
+
+import com.google.gson.Gson;
+import java.util.HashMap;
+import java.util.Map;
+import momento.sdk.auth.accessControl.ExpiresAt;
+import momento.sdk.exceptions.SdkException;
+import momento.token._GenerateDisposableTokenResponse;
+
+public interface GenerateDisposableTokenResponse {
+
+  class Success implements GenerateDisposableTokenResponse {
+    public final String authToken;
+    public final String endpoint;
+    public final ExpiresAt expiresAt;
```

Comment:
if you look at the other java response classes, they all have methods like `valueString()` for accessing the response data. Here, you've exposed these as public properties instead.

I think you probably want to make these properties private and then provide accessor methods. @nand4011 may have thoughts though.